cmd/traffic/cmd/manager/managerutil/envconfig.go

```diff
@@ -61,6 +61,7 @@ type Env struct {
 	AgentEnvoyLogLevel       string                      `env:"AGENT_ENVOY_LOG_LEVEL,    parser=logLevel,       defaultFrom=AgentLogLevel"`
 	AgentEnvoyServerPort     uint16                      `env:"AGENT_ENVOY_SERVER_PORT,  parser=port-number"`
 	AgentEnvoyAdminPort      uint16                      `env:"AGENT_ENVOY_ADMIN_PORT,   parser=port-number"`
+	AgentInjectorName        string                      `env:"AGENT_INJECTOR_NAME,      parser=string,         default=agent-injector"`
```

Comment:
This default is already declared in `values.yaml`. I think we should avoid declaring defaults multiple times. For that reason, I actually removed several of defaults from this file a while back. See my comment in 54011a90.